<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1761</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>marker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/ladybug.png</normaloff>icon/ladybug.png</iconset>
  </property>
  <widget class="QLabel" name="side_label">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>70</y>
     <width>830</width>
     <height>700</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="skip_button">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>880</y>
     <width>210</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
	background-image : url(icon/backb.png); 
	border : 0px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="front_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>830</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/feedback.png</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>819</y>
     <width>1301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 백제 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="background_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>1440</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/b.png</pixmap>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>side_label</zorder>
  <zorder>front_label</zorder>
  <zorder>textBrowser</zorder>
  <zorder>skip_button</zorder>
  <zorder>background_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
